pkg/helper/machinepools/helpers.go

```diff
@@ -221,3 +222,37 @@ func HostedClusterOnlyFlag(r *rosa.Runtime, cmd *cobra.Command, flagName string)
 		os.Exit(1)
 	}
 }
+
+func CreateNodeDrainGracePeriodBuilder(r *rosa.Runtime, nodeDrainGracePeriod string) *cmv1.ValueBuilder {
+	valueBuilder := cmv1.NewValue()
+	if nodeDrainGracePeriod == "" {
+		return valueBuilder
+	}
+
+	nodeDrainParsed := strings.Split(nodeDrainGracePeriod, " ")
+	if len(nodeDrainParsed) > 2 {
+		r.Reporter.Errorf("Invalid value for node drain grace period. Expects only the duration and " +
+			"the unit (minute|minutes|hour|hours).")
+		os.Exit(1)
```

Comment:
@philipwu08 Can you please make this function return an `error` and not `os.Exit(1)` so it can be tested. Thanks!